build.sh

```diff
@@ -11,33 +24,25 @@ else
 fi
 mkdir -p $cachedir
 nugetVersion=latest
-cachePath=$cachedir/nuget.$nugetVersion.exe
+cacheNuget=$cachedir/nuget.$nugetVersion.exe
```

Comment:
Lots of variable renaming. Does this have anything to do with using `KoreBuild,sh`? If not, can it be done separately to make what's really changing more clear?